deployment/release.sh

```diff
@@ -0,0 +1,9 @@
+#  version="USER INPUT"
+read -p "Enter version number: " version
```

Comment:
You know what would be really sweet is if it automatically worked out the version number. You can use `git describe --abbrev=0`

Reply:
I'd rather be able to control it for a few reasons.

- Adds a manual step, so people don't commit to master and release by mistake
- Gives us more control over bumping to major versions
- We may want to skip versions etc

Reply:
I'll print out the current version as an indicator at least!!!